CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.2.2
+
+### Bug fixes
+* Removed forward slash from serialized IPv4/IPv6 columns. NB: IPv4/IPv6 columns are temporarily resolved as `type/TextLike` instead of `type/IPAddress` base type due to an unexpected result in Metabase 0.47 type check.
```

Comment:
Is there an issue to link?

Reply:
Not really; it's just a random commit in master from a huge PR.

Reply:
On 0.47.2, for some reason, it does not want to have a semantic type there which leads to an exception thrown, and `IPAddress` is a semantic type with a `TextLike` base.
The fix (removal of this check) is coming from this PR: https://github.com/metabase/metabase/pull/32657
https://github.com/metabase/metabase/pull/32657/files#diff-ad1b66c2e434ba72459574f419abb6da171fa5c4a1a1ee33a09bb301bef4afb8R53-R54

It was initially introduced in April '23: https://github.com/metabase/metabase/pull/29818/files#diff-ad1b66c2e434ba72459574f419abb6da171fa5c4a1a1ee33a09bb301bef4afb8R41-R43

No idea how Postgres which also uses `IPAddress` type for `inet` database type passes the bundled tests currently on 0.47.x.